### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by mustabbas**
### Change Done
- [ ]  Added error message span and styled 
- [ ] Added function to check email content upper case 
- [ ] Added event listener to listen when submit the form 

### File Changed 
- [ ] index.html
- [ ] style.css
- [ ] script.js

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -51,6 +66,18 @@ function togglePopup(i) {
...
+  if (lowerCase !== email.value) {
+    event.preventDefault();
+    errorSpan.classList.remove('display-none');
+    errorSpan.textContent = 'email should not content upper case';
+  } else {
+    errorSpan.classList.add('display-none');
+    errorSpan.textContent = '';
+  }
+}
+
```
Comment (safafa) [inline assets/script.js:80]: - [ ] Well done on implementing the form validation for the email input. However, you still have one problem to fix. As said in the project requirements 
> if the validation is not OK, you show an error message to the user near the submit button...

When I tried your code the error message appeared near the email input. Kindly fix this issue

- [ ] You're missing [the javascript linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js) kindly add it .

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @mustabbas @DyaryRaoof , You did a great job so far 👏🏼 👏🏼 

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: assets/script.js (hunk 1 above)
Comment (mustabbas) [reply assets/script.js:80]: For the first point project requirements said near but this near no role said what I make not near 
For  the javascript linter not missing I add  that before

Comment (DyaryRaoof) [reply assets/script.js:80]: Mustafa, I think you need to change your linter.yml file to contain the information for all four linters including eslint.

Comment (safafa) [reply assets/script.js:80]: @mustabbas for the linters, you should also modify the linters.yml file in the workflows directory. As you can see you only have 3 checks when you should have 4 the ESlint "JS linter" is missing.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @mustabbas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x]  ESLinter is missing, please add it to this project, feel free to check [This Article](https://github.com/microverseinc/linters-config/tree/master/javascript) to see how to implement ESLinter in your project 😉 

Your current linters:

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @mustabbas, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**